src/pages/ProductList.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import './ProductList.css';
+
+const products = [
+  {
+    "id": 101,
+    "name": "스텐이중식판세트",
+    "price": 10000,
+    "stock": 565,
+    "imageUrl": "https://front-recruitment-task.s3.amazonaws.com/%E1%84%8C%E1%85%AE%E1%84%87%E1%85%A1%E1%86%BC%E1%84%83%E1%85%A9%E1%84%80%E1%85%AE/%EC%A3%BC%EB%B0%A9%EB%8F%84%EA%B5%AC1.jpeg"
+  },
+  {
+    "id": 102,
+    "name": "쟁반 세트",
+    "price": 12900,
+    "stock": 5,
+    "imageUrl": "https://front-recruitment-task.s3.amazonaws.com/%E1%84%8C%E1%85%AE%E1%84%87%E1%85%A1%E1%86%BC%E1%84%83%E1%85%A9%E1%84%80%E1%85%AE/%EC%A3%BC%EB%B0%A9%EB%8F%84%EA%B5%AC2.jpeg"
+  },
+  {
+    "id": 103,
+    "name": "스마일 머그",
+    "price": 15300,
+    "stock": 175,
+    "imageUrl": "https://front-recruitment-task.s3.amazonaws.com/%E1%84%8C%E1%85%AE%E1%84%87%E1%85%A1%E1%86%BC%E1%84%83%E1%85%A9%E1%84%80%E1%85%AE/%EC%A3%BC%EB%B0%A9%EB%8F%84%EA%B5%AC3.jpeg"
+  },
+  {
+    "id": 104,
+    "name": "와인오프너 풀세트",
+    "price": 37000,
+    "stock": 133,
+    "imageUrl": "https://front-recruitment-task.s3.amazonaws.com/%E1%84%8C%E1%85%AE%E1%84%87%E1%85%A1%E1%86%BC%E1%84%83%E1%85%A9%E1%84%80%E1%85%AE/%EC%A3%BC%EB%B0%A9%EB%8F%84%EA%B5%AC4.jpeg"
```

Comment:
이미지 url이 노션 더미데이터에 잘못 기재되어 있던 것 같아요!

```suggestion
    "imageUrl": "https://front-recruitment-task.s3.amazonaws.com/%E1%84%8C%E1%85%AE%E1%84%87%E1%85%A1%E1%86%BC%E1%84%83%E1%85%A9%E1%84%80%E1%85%AE/%E1%84%8C%E1%85%AE%E1%84%87%E1%85%A1%E1%86%BC%E1%84%83%E1%85%A9%E1%84%80%E1%85%AE4.jpeg"
```